### PR TITLE
Fixed unsubscribe error on macos

### PR DESCRIPTION
**Pull request by kj-crypto**

### CODE REVIEW

File: contrib/freeze_packages.sh [hunk 1]
```diff
@@ -6,33 +6,33 @@ set -e
...
-
-
 which virtualenv > /dev/null 2>&1 || { echo "Please install virtualenv" && exit 1; }
 
-
 ${SYSTEM_PYTHON} -m hashin -h > /dev/null 2>&1 || { ${SYSTEM_PYTHON} -m pip install hashin; }
 
-
-for i in '' '-hw' '-binaries' '-wine-build' '-mac-build'; do
+for i in '' '-hw' '-binaries' '-binaries-mac' '-build-wine' '-build-mac' '-build-sdist' '-build-appimage'; do
```
Comment (PJLewandowski) [inline contrib/freeze_packages.sh:23]: Requirements file for windows is still `requirements-wine-build` not `requirements-build-wine`. Is this correct?

File: electrum/gui/qt/email_notification_dialogs.py [hunk 2]
```diff
@@ -635,6 +635,17 @@ def _subscribe(self):
 
     def _unsubscribe(self):
         self.close()
+        is_unsub = custom_message_box(
```
Comment (PJLewandowski) [inline electrum/gui/qt/email_notification_dialogs.py:638]: I know its common for bool vars to starts with `is_` but i dont like it here.

Comment (kj-crypto) [reply electrum/gui/qt/email_notification_dialogs.py]: Done

**Review by PJLewandowski — APPROVED** [review]

File: contrib/freeze_packages.sh (hunk 1 above)
Comment (kj-crypto) [reply contrib/freeze_packages.sh]: Done

**Review by PJLewandowski — APPROVED** [review]

**Pull request merged**